LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
```

Comment:
what does this license grant? why not MIT?

Reply:
This is the weird way that R does licencing :-/

Reply:
I've added a full licence though for clarity as this is a common pain point...